playbook/app/pb_kits/playbook/pb_icon/_icon.scss

```diff
@@ -1,10 +1,18 @@
 @import "../tokens/colors";
 
+$additional_colors: (
+  "default": $text_lt_default,
```

Comment:
I think we can remove default here since this is the same color as the default for the icon kit in general, so doesn't seem needed! The rest is good!